sdk/nodejs/package.json

```diff
@@ -9,7 +9,7 @@
         "directory": "sdk/nodejs"
     },
     "dependencies": {
-        "@grpc/grpc-js": "~1.3.8",
+        "@grpc/grpc-js": "~1.8.11",
```

Comment:
anyone have strong opinions?

Reply:
As in pinning to the exact version? I think we can leave it floating as it was before.